### PR TITLE
Changes for adding version to tables

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4539

*Description of changes:*
- Changes for adding version to tables
- Fixed alembic to include type changes to existing columns
- Adding activity stream option with custom json data with user name and remote address

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/src/auth_api/models/entity.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ class Entity(BaseModel):  # pylint: disable=too-few-public-methods, too-many-ins
     pass_code_claimed = Column('pass_code_claimed', Boolean(), default=False)
     business_number = Column('business_number', String(100), nullable=True)
     name = Column('name', String(250), nullable=True)
-    corp_type_code = Column(String(10), ForeignKey('corp_type.code'), nullable=False)
+    corp_type_code = Column(String(15), ForeignKey('corp_type.code'), nullable=False)
```
Comment (saravanpa-aot) [inline auth-api/src/auth_api/models/entity.py:40]: this change is intentional rite? from column length 10 to 15?

File: auth-api/src/auth_api/models/org_status.py [hunk 2]
```diff
@@ -24,6 +27,11 @@ class OrgStatus(BaseCodeModel):  # pylint: disable=too-few-public-methods # Temp
 
     __tablename__ = 'org_status'
 
+    @declared_attr
+    def code(cls):  # pylint:disable=no-self-argument, # noqa: N805
```
Comment (saravanpa-aot) [inline auth-api/src/auth_api/models/org_status.py:31]: hope this change is also required

File: auth-api/tests/unit/models/test_documents.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def test_documents_with_insert_some_type(session):
     """
     html_content = '<HTML>'
     # putting higher numbers so that version number doesnt collide with existing in db
-    doc = Documents(version_id=20, type='sometype', content=html_content)
+    doc = Documents(version_id=20, type='sometype', content=html_content, content_type='text/html')
```
Comment (saravanpa-aot) [inline auth-api/tests/unit/models/test_documents.py:39]: is this also needed?

Comment (sumesh-aot) [reply auth-api/tests/unit/models/test_documents.py:39]: Yes, at some point someone made the content type as mandatory in model, but the change was not in DB. Now all the model is sync'ed with DB, so we need this change.

File: auth-api/src/auth_api/models/org_status.py (hunk 2 above)
Comment (sumesh-aot) [reply auth-api/src/auth_api/models/org_status.py:31]: This is again same mistake someone did earlier. There were lot of places where the model is not upto what's in DB.

File: auth-api/src/auth_api/models/entity.py (hunk 1 above)
Comment (sumesh-aot) [reply auth-api/src/auth_api/models/entity.py:40]: Please see comments below, base model was changed earlier but not all references were not changed.

File: auth-web/public/config/configuration.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
   "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
+  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
```
[the remark this answers is not in the document]
Comment (sumesh-aot) [reply auth-web/public/config/configuration.json:4]: Its only space :)

File: auth-api/src/auth_api/services/products.py [hunk 5]
```diff
@@ -55,25 +55,32 @@ def create_product_subscription(org_id, subscription_data: Tuple[Dict[str, Any]]
...
 
-                db.session.bulk_save_objects(obj)
-                db.session.commit()
+            Product._create_roles(is_new_transaction, product_code, product_subscription, subscription)
 
         # TODO return something better/useful.may be return the whole model from db
         return subscriptions_model_list
 
+    @staticmethod
+    def _create_roles(is_new_transaction, product_code, product_subscription, subscription):
```
[the remark this answers is not in the document]
Comment (sumesh-aot) [reply auth-api/src/auth_api/services/products.py:65]: Changed the complexity by creating a new function as it got complained as code smell

**Pull request merged**